<!--
   If you want to debug nodes or run the process in a different window:
   http://wiki.ros.org/roslaunch/Tutorials/Roslaunch%20Nodes%20in%20Valgrind%20or%20GDB
-->

<launch>
  <!-- Nodelet Manager Camera Top -->
  <node pkg="nodelet" type="nodelet" name="nodelet_manager_top" args="manager"/>

  <!-- Nodelet Manager Camera Bottom -->
  <node pkg="nodelet" type="nodelet" name="nodelet_manager_bottom" args="manager"/>


   <!-- UVCCam Nodelet Top -->

   <node pkg="nodelet" type="nodelet" name="camTop" args="load cam_nodelet/UVCCamNode nodelet_manager_top">
     <rosparam file="$(find hiwr_camera_controller)/param/config_Microsoft_Lifecam_Cinema.yaml"/>
   </node>


   <!-- UVCCam Nodelet Bottom-->
<!--   <node pkg="nodelet" type="nodelet" name="camBottom" args="load cam_nodelet/UVCCamNode nodelet_manager_bottom">
      <rosparam file="$(find hyve_camera_common)/param/config_Logitech_C920.yaml"/>
   </node>
-->
<!--
   <node pkg="nodelet" type="nodelet" name="bar_code_1" args="load bar_code_nodelet/Bar_code_node nodelet_manager_bottom">
     <param name="video_stream" type="str" value="/camBottom/output_video" />
     <rosparam file="$(find hyve_bar_code_nodelet)/param/config_Bar_Code.yaml"/>
   </node>




   -->
    <node pkg="nodelet" type="nodelet" name="grayscale1" args="load nodelet_webcam_opencv/WebcamGrayscale nodelet_manager_bottom">
        <param name="video_stream" type="str" value="/camBottom/output_video" />
    </node>

</launch>
